samples/get_view_image.py

```diff
@@ -0,0 +1,67 @@
+####
+# This script demonstrates how to use the Tableau Server Client
+# to query a high resolution image of a view from Tableau Server.
```

Comment:
Good catch, download is more appropriate, especially since we are saving the image to a specified filepath